frontend/javascripts/types/api_flow_types.ts

```diff
@@ -217,6 +231,8 @@ export function convertDatasetToCompact(dataset: APIDataset): APIDatasetCompact
     status: dataset.dataSource.status,
     tags: dataset.tags,
     isUnreported: dataset.isUnreported,
+    colorLayerNames: colorLayerNames.sort(),
+    segmentationLayerNames: segmentationLayerNames.sort(),
```

Comment:
```suggestion
    colorLayerNames: colorLayerNames,
    segmentationLayerNames: segmentationLayerNames,
```

I integrated this in my previous comment suggestion.